types/cavy/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for cavy 0.6
+// Project: https://github.com/pixielabs/cavy
+// Definitions by: Tyler Hoffman <https://github.com/tyler-hoffman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import * as React from 'react';
+
+export {};
+
+type RefCallback = (element: React.ReactNode | null) => void;
+
+export type TestHookGenerator = (label: string, callback?: RefCallback) => RefCallback;
+
+export type WithTestHook<T extends {}> = T & { generateTestHook: TestHookGenerator };
+
+export function hook<T extends {}>(component: React.ComponentClass<WithTestHook<T>>): React.ComponentClass<T>;
+
+export interface TesterProps {
+    specs: Array<(spec: TestScope) => void>;
+    waitTime: number;
+    sendReport?: boolean;
```

Comment:
`Property 'store' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Tester> & Readonly<TesterProps> & Readonly<{ children?: ReactNode; }>'`